apps/src/utils.js

```diff
@@ -1,5 +1,6 @@
-/* global define, $ */
-import Immutable from 'immutable';
+// @flow
```

Comment:
Would we start including this at the top of every file we want flow to look at?  Can this be scoped?  Eventually, I assume we can remove these and enable for all files under /src or something?  Why don't we do that right away, since code missing types is flow-compatiable?